modules/serverless-configuring-event-delivery-examples.adoc

```diff
@@ -19,7 +19,7 @@ spec:
   delivery:
     deadLetterSink:
       ref:
-        apiVersion: eventing.knative.dev/v1alpha1
+        apiVersion: eventing.knative.dev/v1beta1
```

Comment:
Only operator.knative.dev (KnativeServing,KnativeEventing) should be changed. Not `eventing.knative.dev`